argas7/learning_flutter
# Feat/create project

### What I Did

- Create the project with flutter create app command
- Create docs folder to documentate first steps for the project

### What I Learned

- How to create a new flutter project
- How to list all connected devices

### How to test

- Run the project with flutter run command
- Check if the project is running on the device
